packages/Molecules/Form/fields/fieldRplselect.vue

```diff
@@ -1,118 +1,384 @@
 <template>
-  <multiselect
-    :options="options"
-    :value="localValue"
-    v-bind="selectOptions"
-    :placeholder="schema.placeholder"
-    :custom-label="customLabel"
-    :max="schema.max || null"
-    :disabled="disabled"
-    @input="updateSelected"
-    @search-change="onSearchChange"
-    @tag="addTag"
-    :option-height="selectOptions.optionHeight"
-  >
-  </multiselect>
+  <div class="rpl-select" :class="{'rpl-select--open' : isOpen}">
+    <div v-if="$breakpoint.s" class="rpl-select__native">
+      <select :multiple="schema.multiselect" v-model="value">
+        <option v-if="!schema.multiselect" disabled value="">{{placeholder}}</option>
+        <option :value="option.id" v-for="(option) in options" :key="option.id">{{option.name}}</option>
+      </select>
+      <rpl-icon class="rpl-select__trigger-icon" symbol="down" color="primary" />
+    </div>
+    <div class="rpl-select__inner">
+      <div
+        class="rpl-select__trigger"
+        tabindex="0"
+        aria-haspopup="listbox"
+        :aria-labelledby="`${schema.model}-rpl-select-trigger ${schema.model}-rpl-select-value`"
+        :aria-expanded="isOpen ? 'true' : false"
+        role="button"
+        ref="trigger"
+        @click="toggleOpen"
+        v-on:keyup.enter.prevent="toggleOpen"
+      >
```

Comment:
Using NVDA on windows 10 with IE11 and Edge, I found that IE11 wouldn't read out the single select options as you press arrow up and down (so a user wouldn't know what to select), and on Edge, single select DID work, but multiselect did not.

Reply:
wasn't managing focus, in my testing it now reads options in both NVDA and voiceover